examples/veins/RSUExampleScenario.ned

```diff
@@ -19,7 +19,4 @@ import org.car2x.veins.nodes.Scenario;
 network RSUExampleScenario extends Scenario
 {
     submodules:
-        rsu[1]: RSU {
```

Comment:
Why are you removing the RSU from the RSU scenario?

Reply:
I suspect that's a mistake! I suppose that's why you didn't merge it yet, @sommer? I'll see if I can find some time to test this stuff properly this week.

Reply:
Ah, I see why now -- the code crashes if you run it with an RSU:
```
<!> ASSERT: Condition '!std::isnan(senderHeight)' does not hold in function 'calculateVehicleAttenuation' at veins/modules/obstacle/ObstacleControl.cc:467 -- in module (Veins::ObstacleControl) RSUExampleScenario.obstacles (id=2), at t=0.602766975009s, event #13
```
I'm pretty sure the RSU size is not set correctly, I'm having a look whether there is an easy fix.

Reply:
fixing this properly will require some code that tells the `ObstacleControl` that the RSUs exist (by using `addFromTypeAndShape` or `addFromXml`). The problem is that RSUs don't have any connection with the TraCI mobility (because RSUs don't exist in SUMO), where currently the obstacles and vehicles are submitted. I'm not sure whether it's easier to add the RSUs to this process by setting their mobility as `TraCIMobility`, or to modify the `BaseMobility` code to get this to work.

Reply:
The Vehicle Obstacle Shadowing model currently uses hard-coded heights for checking whether something intersects the line of sight vertically. It assumes a fixed height for vehicles, so that is (partly) sorted. It has no idea how to deal with RSUs. Surely they are transmitting from a certain height, so this would be an easy fix; right now, the model just complains about a missing value...